bin/composer_paths.sh

```diff
@@ -53,7 +53,7 @@ echo "::debug::${composer_version}"
 echo "::debug::Composer cache directory found at '${cache_dir}'"
 echo "::debug::File composer.json found at '${composer_json}'"
 echo "::debug::File composer.lock path computed as '${composer_lock}'"
-echo "::set-output name=command::${composer_path}"
-echo "::set-output name=cache-dir::${cache_dir}"
-echo "::set-output name=json::${composer_json}"
-echo "::set-output name=lock::${composer_lock}"
+echo "command=${composer_path}" >> "${GITHUB_OUTPUT}"
```

Comment:
Shellcheck doesn't like this output being called `command`...

```
In bin/composer_paths.sh line 56:
echo "command=${composer_path}" >> "${GITHUB_OUTPUT}"
^-- SC2129: Consider using { cmd1; cmd2; } >> file instead of individual redirects.

For more information:
  https://www.shellcheck.net/wiki/SC2129 -- Consider using { cmd1; cmd2; } >>...
```

Reply:
I've changed the variable name to `composer_command`.

It doesn't seem like this will have any backwards compatibility concerns since it's only used within the `action.yml` file. But not sure if I'm missing something.

Reply:
I'd need to check, but this output _may_ also be available to steps in external workflows being run _after_ the step using this action runner.
If that's the case and while it would be rare for a workflow to use the `command` output, it would still need a changelog entry. Though as I said, this would need to be checked and confirmed first.